runtime/builtin_types.go

```diff
@@ -541,6 +541,18 @@ func builtinRepr(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
 	return s.ToObject(), nil
 }
 
+func builtinSorted(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
+	if raised := checkFunctionArgs(f, "sorted", args, ObjectType); raised != nil {
```

Comment:
`sorted()` is a little more complicated than this:

```
sorted(iterable, cmp=None, key=None, reverse=False) --> new sorted list
```

A TODO is fine. Then that functionality could be added to list.sort() and then passed when you call list.sort() below.